gitCommit.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Commit changes form for easy commits
+
+read -rp "Enter your working branch name: " branch
+read -rp "Enter your task type: " task
+read -rp "Enter the task scope: " scope
+read -rp "Enter the task summary: " summary
+read -rp "Enter the task description: " description
+read -rp "What issue number does this address? " issue
+
+git add .
+git commit -m "$task($scope): $summary \n $description \n closes #$issue"
+git pull
+git push
+
+git checkout the-one
```

Comment:
this breaks the logic, because you will be on the-one afterwards - are you sure you didn't just want to fetch the-one?

Reply:
In what what does this break the logic?

All of these changes affect the developer's fork, not the original. So what I am doing with this is apply changes from the local working branch (`dev-cypress` in my case) to the local `the-one` from the developer's fork. I added a pull after changing to the `fork: the-one`. That can potentially break the logic.

Reply:
The script works, I have verified it several times. Barring an explanation of how the script is broken, it is as it is intended to be.